src/classes/TDTM_Filter.cls

```diff
@@ -491,11 +491,11 @@ public class TDTM_Filter {
     * @param filteredList The resulting filtered list.
     * @return void 
```

Comment:
I think we need to add new param here since we add the new parameter in this function.